voting-contract/README.md

```diff
@@ -0,0 +1,5 @@
+# Voting Contract
+
+The purpose of this contract is for people to vote on the next reset height for the network that the
+contract is deployed on. Anyone can submit a proposal but only stakeholders can vote. Once over 2/3 of total
+stake in an epoch has voted on a proposal, the proposed height becomes final and the voting is closed.
```

Comment:
So I thought we are going to do next thing:
 - Proposal are sent with date of unlock all the time, there is a window to vote for proposal is time to unlock - 2 weeks - 4 epochs
 - Validators vote for proposals as they see fit until voting is allowed: either yay or nay
 - Proposal voting locks for 4 epochs and at that point we evaluate who voted for it, and what is their stake. If 66% of total stake voted -> proceed on 2 week waiting period before unlocking.

Why 4 epochs? Because we want to allow delegates to switch their validators if they don't think the validator they are delegating right now represents them.

Reply:
Why do we need 2 week waiting period?

Are you commenting on upgradability or transfers vote?

Reply:
Let's move this discussion to https://github.com/nearprotocol/NEPs/pull/62

Reply:
Moved there.